java/src/main/java/com/plaid/quickstart/resources/PaymentInitiationResource.java

```diff
@@ -26,35 +26,40 @@
 public class PaymentInitiationResource {
   private static final Logger LOG = LoggerFactory.getLogger(PaymentInitiationResource.class);
 
-  private final PlaidClient plaidClient;
+  private final PlaidApi plaidClient;
 
-  public PaymentInitiationResource(PlaidClient plaidClient) {
+  public PaymentInitiationResource(PlaidApi plaidClient) {
     this.plaidClient = plaidClient;
   }
 
   @GET
   public PaymentResponse getPayment() throws IOException {
     String paymentId = QuickstartApplication.paymentId;
 
-    Response<PaymentGetResponse> paymentGetResponse =
-      plaidClient.service().paymentGet(new PaymentGetRequest(paymentId)).execute();
+    PaymentInitiationPaymentGetRequest payRequest = new PaymentInitiationPaymentGetRequest()
+      .paymentId(paymentId);
+
+    Response<PaymentInitiationPaymentGetResponse> paymentGetResponse =
+      plaidClient
+      .paymentInitiationPaymentGet(payRequest)
+      .execute();
     if (!paymentGetResponse.isSuccessful()) {
       try {
-        ErrorResponse errorResponse = plaidClient.parseError(paymentGetResponse);
-        LOG.error("error: " + errorResponse);
+        Gson gson = new Gson();
+          Error errorResponse = gson.fromJson(paymentGetResponse.errorBody().string(), Error.class);
+          System.out.println(errorResponse);
```

Comment:
is there a reason we swapped `LOG.error` for `System.out.println`?

Reply:
nope. changed back